doc/ViamSDK.md

```diff
@@ -0,0 +1,3 @@
+# Viam SDK Library
+
+Robots at your fingertips.
```

Comment:
Open to alternatives but we shouldn't add more occurances of the term robot
```suggestion
Work with your machines and hardware.
```

Reply:
Not an additional occurrence but a copy from the README. 

But you are correct we shouldn't be using robot anymore. This was a pun since the flutter SDK is about connecting your smart phone to machines. I would like it if we could remain punny but obvi don't want to spend too much time on coming up with a new one

Maybe "Smart machines on smart devices"??